pkg/flags/flags.go

```diff
@@ -55,6 +55,7 @@ const (
 	ServiceRefFlagName    = "--service-ref"
 	SinceFlagName         = "--since"
 	SourceImageFlagName   = "--source-image"
+	SubPathFlagName       = "--subpath"
```

Comment:
Sub path is two words

```suggestion
	SubPathFlagName       = "--sub-path"
```